internal/cmd/gocdk/serve.go

```diff
@@ -239,6 +239,7 @@ loop:
 }
 
 // buildForServe runs Wire and `go build` at moduleRoot to create exePath.
+// Note that on Windows, exePath must end with .EXE.
 func buildForServe(ctx context.Context, pctx *processContext, moduleRoot string, exePath string) error {
```

Comment:
Could also modify `exePath` here as needed, and add it as a return value. It's a bit awkward for the existing callers since they stash `exePath` in a struct.